static/app/views/replays/list/replays.tsx

```diff
@@ -92,7 +92,7 @@ function ReplaysList() {
         ) : (
           <ReplayOnboardingPanel>
             <Button onClick={activateSidebar} priority="primary">
-              {t('Get Started')}
+              {t('Set Up Replays')}
```

Comment:
Replays? or Session Replay?

Reply:
Our left nav is Replays so that's why I went with that (and it's shorter). But either works!